Makefile

```diff
@@ -585,8 +587,18 @@ rpm: tarball test
 
 # Removing test dependency until resolved
 tarball:
-	VERSION=`awk '/^Version/ {print $$2}' deepsea.spec`; \
-	git archive --prefix deepsea-$$VERSION/ -o ~/rpmbuild/SOURCES/deepsea-$$VERSION.tar.bz2 HEAD
+	mkdir -p .tmp
+	git archive --prefix deepsea-tmp/ -o .tmp/deepsea-tmp.tar HEAD
+	HASH=`git rev-parse --short HEAD`; \
+	OFFSET=`git rev-list \`git log --pretty=oneline -- version.txt | head -1 | cut -d ' ' -f 1\`..HEAD | wc -l`; \
```

Comment:
@jan--f This line is key, I think. Instead of using `git describe`, @rjfd is calculating the offset as the number of commits since the last commit that changed version.txt.